tests/unit/database/test_algorithm_service.py

```diff
@@ -15,34 +15,30 @@
 
 
 def test_list_algorithms(session):
-    algo1 = models.Algorithm(label="basic", is_default=True, description="First algorithm")
```

Comment:
I know the naming is a little misleading, but these tests are scoped around the API endpoints and how they return data rather than the content of the algorithm.  So for this first one, its just showing that if 2 algorithms are in the database, the listing method returns info on both of them.

Reply:
Ah got it, that makes sense. I'll add that one back in since we do want to test on returning multiple algorithms in case the user uploads one of their own.

Reply:
Honestly, it's not a big deal.  The tests still function as expected with the changes.  I'm just bringing it up to give context as to what the test was focusing on.  I probably should have prefaced that comment with a "nit:" or something other label to indicate I was adding context rather that requesting a change.